src/d3a/models/market/market_structures.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, id, price, energy, seller, original_offer_price=None, seller_
         self.energy = energy
         self.seller = seller
         self.seller_origin = seller_origin
+        self.energy_rate = price / energy
```

Comment:
Could be used [here](https://github.com/gridsingularity/d3a/pull/675/files#diff-286711868906f1b84a065750ab4e674fR40) and [here](https://github.com/gridsingularity/d3a/pull/675/files#diff-286711868906f1b84a065750ab4e674fR45) ?

Reply:
[here](https://github.com/gridsingularity/d3a/pull/675/files#diff-286711868906f1b84a065750ab4e674fR68) too

Reply:
I cannot see where the links refer to, but I did a global search and replaced the division with the energy_rate parameter. Thanks